pyproject.toml

```diff
@@ -23,7 +23,7 @@ python = "^3.7.1"
 jupyter-dash = ">=0.4.2"
 plotly = "^5.5.0"
 dash = "^2.2.0"
-pandas = "^1.3.5"
+pandas = "*"
```

Comment:
We might want to lock on at least major version, unless we explicitly support any.

Reply:
Indeed, as for now, we've thoroughly tested 1*, so maybe lock-in on that version, what do you think @jvdd?

Reply:
Is fine for me! We want however add support for pandas 2 as well..
`>= 1` is fine for me?